test/testQuitDriverAfterEachIteration.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
@aniljca does this test fail without your change? I notice there are no "click" events, so not sure it would actually encounter the problem.